shared/src/libs/helpers/helpers.ts

```diff
@@ -1,4 +1,5 @@
 export { configureString } from './configure-string/configure-string.helper.js';
 export { getFormattedDate, getFormattedTime } from './date/date.js';
 export { debounce } from './debounce/debounce.helper.js';
+export { getMegabytesFromBytes } from './get-megabytes-rom-bytes/get-megabytes-from-bytes.helper.js';
```

Comment:
The typo is in the folder name itself, not in the import
```suggestion
export { getMegabytesFromBytes } from './get-megabytes-from-bytes/get-megabytes-from-bytes.helper.js';
```

Reply:
Done